recon_test_pack/PET_ACQ_small.l.hdr.STIR

```diff
@@ -0,0 +1,67 @@
+!INTERFILE :=
+!originating system :=2008
+%SMS-MI header name space :=PETLINK bin address
+%SMS-MI version number :=3.4
+!GENERAL DATA :=
+!data offset in bytes :=0
+name of data file := small_listmode_file.l
+!GENERAL IMAGE DATA :=
+!type of data := PET
+number of bytes per pixel := 4
+!type of data :=PET
+%study date (yyyy:mm:dd) :=2014:06:24
+%study time (hh:mm:ss GMT+00:00):=15:45:07
+isotope name :=F-18
+isotope gamma halflife (sec) :=6586.2
+isotope branching factor :=0.97
+radiopharmaceutical :=Fluorodeoxyglucose
+relative time of tracer injection (sec) :=0
+tracer activity at time of injection (Bq) :=6e+007
+injected volume (ml):=0
+%tracer injection date (yyyy:mm:dd):=2014:06:24
+%tracer injection time (hh:mm:ss GMT+00:00):=15:14:00
+%patient orientation :=HFS
+PET data type :=Emission
+data format :=CoincidenceList
+horizontal bed translation :=stepped
+start horizontal bed position (mm) :=0
+end horizontal bed position (mm) :=0
+start vertical bed position (mm) :=0
+%bed zero offset (mm) :=0
+number of energy windows :=1
+%energy window lower level (keV) [1]:=430
+%energy window upper level (keV) [1]:=610
+!PET STUDY (Emission data) :=
+PET scanner type :=cylindrical
+transaxial FOV diameter (cm) :=59.6
+number of rings :=64
+distance between rings (cm) :=0.40625
+gantry tilt angle (degrees) :=0
+gantry crystal radius (cm) :=32.8
+bin size (cm) :=0.20445
+septa state :=none
+%number of TOF time bins:=1
+%TOF mashing factor:=1
+!IMAGE DATA DESCRIPTION :=
+%preset type :=time
+%preset value :=3600
+%preset unit:=seconds
+number of time frames:=1
+!image duration (sec)[1] :=3600
+%total listmode word counts :=1210129934
+%COINCIDENCE LIST DATA :=
+%LM event and tag words format (bits) :=32
+%timing tagwords interval (msec) :=1
+%singles polling method :=instantaneous
+%singles polling interval (sec) :=2
+%singles scale factor :=8
+%total number of singles blocks :=224
+%axial compression :=1
+%maximum ring difference :=60
+%number of projections :=344
+%number of views :=252
+%number of segments :=11
+%segment table :={127,115, 115, 93, 93 ,71, 71, 49, 49, 27, 27}
```

Comment:
I'm worried about this. It implies that the list file you're using was using axial compression. Currently we're not reading this information (Work-in-progress). So, it has to mean that the data is actually interpreted wrongly. Now in principle that doesn't matter for the test (as we just check consistency between list- and proj-recon), but if anyone every checks the reconstructed images (or sinogram output), they're bound to be crazy.

Reply:
The test probably doesn't care what data is in there as it just checks consistency between the recons (not if the recons make sense). As I wrote, I would guess that currently they don't.

I would prefer though to either have a listfile without axial compression (would have to be from the mMR), or to fix the listmode code to take the axial compression into account (probably not a 1 minute fix...), or is this fixed in the latest commit by using `InterfilePDFSHeader` for the parsing?